<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Artist</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/icon.jpg</normaloff>:/res/icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>91</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="ButtonSettings">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/gear.ico</normaloff>:/res/icons/gear.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonStartStop">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/play.ico</normaloff>:/res/icons/play.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonCapture">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/camera.ico</normaloff>:/res/icons/camera.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonDraw">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/paintbrush2.ico</normaloff>:/res/icons/paintbrush2.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="midLineWidth">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonLoad">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/upload.ico</normaloff>:/res/icons/upload.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonQuit">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/res/icons/power.ico</normaloff>:/res/icons/power.ico</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ImageView">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(79, 93, 115);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ButtonSettings</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonSettingsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonStartStop</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonStartStopClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonCapture</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonCaptureClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonDraw</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>ButtonDrawClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>55</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ButtonSettingsClicked()</slot>
  <slot>ButtonDrawClicked()</slot>
  <slot>ButtonCaptureClicked()</slot>
  <slot>ButtonStartStopClicked()</slot>
 </slots>
</ui>
